mimecast/assets/dashboards/mimecast_ttp_url.json

```diff
@@ -0,0 +1,748 @@
+{
+    "title": "Mimecast - TTP URL",
+    "description": "This log shows the details of events which were generated and flagged by TTP URL configuration because the user clicks on a malicious link.",
```

Comment:
```suggestion
    "description": "This log shows the details of events which were generated and flagged by TTP URL configuration because the user clicked on a malicious link.",
```